internal/oci/oci_linux.go

```diff
@@ -25,6 +27,13 @@ func (r *runtimeOCI) createContainerPlatform(c *Container, cgroupParent string,
 		},
 	}
 
+	// First, set the cpuset as the one for the infra container.
+	// This should be overridden if specified in a workload.
+	if r.config.InfraCtrCPUSet != "" && r.config.ConmonCgroup == utils.PodCgroupName {
```

Comment:
Why we need the second check?

Reply:
updated with a comment--it's possible a user has configured the cpuset of the conmon cgroup